include/picongpu/particles/ParticlesFunctors.hpp

```diff
@@ -90,10 +90,12 @@ namespace picongpu
             HINLINE void operator()(const std::shared_ptr<T_DeviceHeap>& deviceHeap) const
             {
 #if(BOOST_LANG_CUDA || BOOST_COMP_HIP)
+                eventSystem::getTransactionEvent().waitForFinished();
```

Comment:
before we used the stream zero which is a blocking stream in CUDA and cupla emulates the same behavior.

I will remove it because line94 the notifying the event properly and therefore a explicit synchronization is not required.